lib/hmc5883l.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * HMC5883L I2C magnetometer driver
+ * http://cylonjs.com
+ *
+ * Copyright (c) 2014-2016 The Hybrid Group
+ * Licensed under the Apache 2.0 license.
+*/
+/*eslint no-undef: "error"*/
+/*eslint-env node*/
+
+"use strict";
+
+var Cylon = require("cylon");
+
+var I2CDriver = require("./i2c-driver");
+
+/**
+ * HMC5883L Driver
+ *
+ * @constructor hmc5883L
+ */
+
+var HMC5883L = module.exports = function HMC5883L() {
+  HMC5883L.__super__.constructor.apply(this, arguments);
+  this.address = this.address || 0x02;
```

Comment:
Shouldn't this be `0x1e` here? https://github.com/jarzebski/Arduino-HMC5883L/blob/master/HMC5883L.h#L30